http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/SamAuthProvider.scala

```diff
@@ -251,12 +251,40 @@ class SamAuthProvider[F[_]: OpenTelemetryMetrics](
           F.raiseError(
             AuthProviderException(
               traceId,
-              s"[SamAuthProvider.lookupOriginatingUserEmail] User ${samUserInfo.userEmail.value} is disabled.",
+              s"[SamAuthProvider.lookupOriginatingUserEmail] User ${samUserInfo.userEmail.value} is disabled",
               StatusCodes.Forbidden
             )
           )
     } yield samUserInfo.userEmail
 
+  /**
+   * Confirm the Sam user is enabled (accepted terms of service etc).
+   * Raises an AuthProviderException if user does not exist or is disabled.
+   */
+  override def checkUserEnabled(petOrUserInfo: UserInfo)(implicit ev: Ask[F, TraceId]): F[Unit] =
```

Comment:
We should probably have a test in the SamAuthProviderSpec that tests this code.

Reply:
@n8landolt done!